chainerx_cc/chainerx/native/CMakeLists.txt

```diff
@@ -59,6 +60,14 @@ if(${BLAS_FOUND})
     endif()
 endif()
 
+# LAPACK
+if(${LAPACK_FOUND})
+MESSAGE( STATUS "LAPACK is found, defining CHAINERX_ENABLE_LAPACK variable.")
+add_definitions(-DCHAINERX_ENABLE_LAPACK=${LAPACK_FOUND})
+include_directories(${LAPACK_INCLUDE_DIRS})
+target_link_libraries(chainerx_native ${LAPACK_LIBRARIES})
+endif()
```

Comment:
Can you add a level of indentation to this if scope?

Reply:
Done.